lib/telegraf/telegraf.conf.erb

```diff
@@ -572,7 +573,9 @@
   ##    root@tcp(127.0.0.1:3306)/?tls=false
   #
   ## If no servers are specified, then localhost is used as the host.
-  servers = ["root@tcp(127.0.0.1:3306)/"]
+  servers = <%= config_file['mysql'] %>
+  tagexclude = ["server"]
```

Comment:
I agree. I'll post in the basecamp todo asking about metric naming here and update it once we decide.